src/core/client/admin/routes/Stories/StoryTable.tsx

```diff
@@ -31,78 +33,93 @@ interface Props {
   multisite: boolean;
 }
 
-const StoryTable: FunctionComponent<Props> = (props) => (
```

Comment:
These changes look deceptively large, I mostly just added some state hooks before the return statement and it changed indentation.